Dalamud/Game/ClientState/Aetherytes/AetheryteEntry.cs

```diff
@@ -62,7 +62,7 @@ internal AetheryteEntry(TeleportInfo data)
     /// <summary>
     /// Gets a value indicating whether this Aetheryte is an Appartment or not.
     /// </summary>
-    public bool IsAppartment => this.data.IsAppartment;
+    public bool IsAppartment => this.data.IsApartment;
```

Comment:
Please fix the spelling here.

Reply:
Am I correct in presuming you want it to be:
```cs
public bool IsApartment=> this.data.IsApartment;
```